protected/components/ModelSearch.php

```diff
@@ -356,9 +356,9 @@ public function retrieveResults()
     /**
      * Add a filter for active
      */
-    public function addActiveFilter()
+    public function addActiveFilter($active_label = null)
     {
-        $this->addSearchItem('active', array('type' => 'boolean'));
+        $this->addSearchItem($active_label==null?'active':$active_label, array('type' => 'boolean'));
```

Comment:
$active_label==null can probably be condensed to simply $active_label. In a ternary expression a variable in the condition section is treated as a boolean so it should function the same as the equality check you have here.

Alternatively, you could use the ?: operator to shrink this even further ie. $active_label ?: 'active'.